src/config.cpp

```diff
@@ -1257,17 +1257,25 @@ namespace config {
       BOOST_LOG(fatal) << "Failed to apply config: "sv << err.what();
     }
 
-    if (!config_loaded) {
 #ifdef _WIN32
+    // UCRT64 raises an access denied exception if launching from the shortcut
+    // as non-admin and the config folder is not yet preset; we can defer
```

Comment:
```suggestion
    // as non-admin and the config folder is not yet present; we can defer
```

Should this be present?